### PR TITLE
swap order on version for consistency

### PR DESCRIPTION
**Pull request by alfredoihernandez**
## Type of PR

- [ ] Documentation changes
- [x] Code changes
- [x] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

### Validation
- [x] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above
 
### Issues Closed or Referenced

- Closes #<issue number> (this will automatically close the issue when the PR closes)
- References #<issue number> (this references the issue but does not close with PR)

### CODE REVIEW

File: TestFiles/baseline.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [
-  {"path":"/version","perfTarget":{"category":"Static"},"validation":{"jsonObject":[{"field":"apiVersion","value":"1.0"},{"field":"appVersion"}]}},
```
Comment (bartr) [inline TestFiles/baseline.json:2]: Order of the test doesn't matter

File: src/app/Middleware/version.cs [hunk 2]
```diff
@@ -46,7 +46,7 @@ public static IApplicationBuilder UseVersion(this IApplicationBuilder builder)
                         }
 
                         // build and cache the json string
-                        string json = "{ " + $"\"apiVersion\": \"{swaggerVersion}\", \"appVersion\": \"{Middleware.VersionExtensions.Version}\"" + " }";
+                        string json = "{ " + $"\"appVersion\": \"{Middleware.VersionExtensions.Version}\", \"apiVersion\": \"{swaggerVersion}\"" + " }";
```
Comment (bartr) [inline src/app/Middleware/version.cs:49]: the order seems inverted to me - apiVersion sorts above appVersion and apiVersion is common across all languages whereas appVersion isn't.  What's driving the change?

**Review by bartr — APPROVED** [review]
approved with comments

File: src/app/Middleware/version.cs (hunk 2 above)
Comment (alfredoihernandez) [reply src/app/Middleware/version.cs:49]: java and typescript repos are set to app and then api, so just consistency

Comment (alfredoihernandez) [reply src/app/Middleware/version.cs:49]: cancelling PR and making change to java instead

**Comment by alfredoihernandez** [comment]
Making changes to helium-java instead for consistency

**Pull request closed without merging**